docs/next.config.js

```diff
@@ -130,4 +130,8 @@ module.exports = {
     });
     return pathMap;
   },
+  async headers() {
+    const cacheHeaders = [{ key: 'Cache-Control', value: 'public, max-age=15552000, immutable' }];
```

Comment:
Any reason to put 6 months of caching?

Reply:
It's a long time! We can also put the maximum supported value `31536000` https://web.dev/http-cache/#versioned-urls

Reply:
In my mind, 6 months seemed long, but looking at other docs sites, looks like they have 31536000. So, I think 6 months is fine!